### DISCUSSION TITLE
where  CONFIG_DMA_GW is enabled ?

### DISCUSSION THREAD

**Discussion by DINESHKUMARK1** [thesofproject/sof#7572] [General]
hi ,
I was looking for  where CONFIG_DMA_GW is enabled but could'nt find in any of the platforms specific deconfig files, could anyone please let me know CONFIG_DMA_GW used or not in any of the sof supported platform.

**Comment by paulstelian97** [comment]
I have found it used by the Tigerlake platform. It used to be used on other platforms, however those were deprecated and are no longer supported on the main branch of the firmware (where it's active development)

The file src/platform/Kconfig has it SELECTed under the CONFIG_TIGERLAKE configuration.

**Comment by ghost** [comment]
Can someone help me how to repair this sound issue on sof-apl-es88336? I do really want to change operating system but I don't know how to deal with this one yet :<
![IMG_20230508_074832_765](https://user-images.githubusercontent.com/114453648/236991990-e0954a59-ee82-4026-b250-26fb118f7cb9.jpg)

**Comment by lgirdwood** [comment]
Not seen any other updates so assuming fixed, this looks like the FW and topology files are not installed on host FS.
